Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -2027,6 +2045,683 @@ script:
       type: string
   - description: Use this command to unstuck the fetch stream in case it's getting duplicated incidents.
     name: prisma-cloud-compute-unstuck-fetch-stream
+  - name: prisma-cloud-compute-ci-scan-results-list
+    description: Retrieves all scan reports for images scanned by the Jenkins plugin or twistcli. Maps to Monitor > Vulnerabilities > Images > CI in the Console UI. The default will retrieve only the passed scans.
+    arguments:
+    - name: account_ids
+      description: A comma-separated list of cloud account IDs to filter the result by.
+      isArray: true
+    - name: resource_ids
+      description: A comma-separated list of resource IDs to scope the query by.
+      isArray: true
+    - name: region
+      description: A comma-separated list of regions to scope the query by.
+      isArray: true
+    - name: scan_id
+      description: Scan ID used in the image layers fetch.
+    - name: image_id
+      description: Image ID of scanned image.
+    - name: job_name
+      description: A comma-separated list of Jenkins job names.
+      isArray: true
+    - name: search
+      description: Retrieves the result for a search term.
+      isArray: true
+    - name: pass
+      description: Indicates whether to filter on passed scans (true) or not (false).
+      defaultValue: "true"
+      predefined:
+      - "true"
+      - "false"
+    - name: scan_time_to
+      description: Filters results by end datetime. Based on scan time.
+    - name: scan_time_from
+      description: Filters results by start datetime. Based on scan time.
+    - name: limit
+      description: The maximum number of CI scan results to return. Must be between 1-50.
+      defaultValue: "50"
+    - name: offset
+      description: The offset by which to begin listing CI scan results.
+      defaultValue: "0"
+    - name: all_results
+      description: Whether to retrieve all results. The "limit" argument will be ignored. Might slow down the command run time. Using this argument may return a lot of results and is not recommended to be used often.
```

Comment:
@ShirleyDenkberg please review this description.

Reply:
I asked Shirley about it on Slack, she approved this.